test/multiverse/suites/rack/puma_rack_builder_test.rb

```diff
@@ -81,7 +81,7 @@ def test_puma_rack_builder_is_auto_instrumented
           ["Middleware/Rack/PumaRackBuilderTest::MiddlewareTwo/call", "Controller/Rack/PumaRackBuilderTest::ExampleApp/call"],
           ["Nested/Controller/Rack/PumaRackBuilderTest::ExampleApp/call", "Controller/Rack/PumaRackBuilderTest::ExampleApp/call"]
         ],
-        :ignore_filter => /^(Supportability|RubyVM|Memory|CPU|Logging|Proc)/
+        :ignore_filter => /^(Supportability|RubyVM|Memory|CPU|Logging)|(\/Rack\/Proc\/)/
```

Comment:
perhaps:

```diff
- :ignore_filter => /^(Supportability|RubyVM|Memory|CPU|Logging)|(\/Rack\/Proc\/)/
+ ignore_filter: %r{^(?:Supportability|RubyVM|Memory|CPU|Logging|/Rack/Proc)}
```

Reply:
Hmm, I don't think that would work, because the first grouping is specifying start of string, which doesn't work for the rack/proc metrics. Rack/Proc is in the middle of the metrics that need to be ignored, which is why i have it in a separate grouping from the ones that are beginning the string

Reply:
Ah, my apologies for missing the caret applying only to the first set of parens.

Reply:
I didn't notice it the first time either! sneaky in there. Thats why my first fix for this issue didn't actually work lol